libs/blocks/notification/notification.js

```diff
@@ -122,21 +127,29 @@ async function decorateLockup(lockupArea, el) {
   if (pre && pre[2] === 'icon') el.classList.replace(pre[0], `${pre[1]}-lockup`);
 }
 
+async function decorateForegroundText(el, container) {
+  const text = container?.querySelector('h1, h2, h3, h4, h5, h6, p')?.closest('div');
+  text?.classList.add('text');
+  const iconArea = text?.querySelector('p picture')?.closest('p');
```

Comment:
This feels a bit weird with targeting a sibling to then target its parent. Could we get away with `text?.querySelector('p:has(picture)')`?

Reply:
Ah, so we can now! Safari had been holding us back from using `:has()` since it wasn't supported until 15.4, but looks like our [browser support matrix](https://wiki.corp.adobe.com/pages/viewpage.action?spaceKey=WP4&title=Test+Automation+Framework#TestAutomationFramework-Browser&PlatformSupportMatrix) has been updated 🥳